tests/python/gaia-ui-tests/gaiatest/apps/camera/app.py

```diff
@@ -59,6 +81,21 @@ def take_photo(self):
         # Wait for thumbnail to appear
         self.wait_for_thumbnail_visible()
 
+    def take_photo_with_timer(self):
+        # Wait for camera to be ready to take a picture
+        controls = self.marionette.find_element(*self._controls_locator)
+        Wait(self.marionette, timeout=20).until(
+            lambda m: controls.get_attribute('data-enabled') == 'true')
+         
+        self.tap_capture()
+        time.sleep(1)
```

Comment:
I know you added time.sleep() for timing issues you saw, but we should try to figure out wait statements that we can rely on to avoid sleep() calls